docker/rumble/Dockerfile

```diff
@@ -0,0 +1,34 @@
+FROM cern/cc7-base:20200504-1.x86_64
+MAINTAINER Ceyhun Uzunoglu ceyhunuzngl@gmail.com
+
+ADD hadoop.repo /etc/yum.repos.d/hadoop.repo
+
+# hadoop related RPMs
+RUN yum install -y cern-hadoop-config spark-bin-3.0 hadoop-bin-2.7.5 java-1.8.0-openjdk-devel gcc git
+RUN yum clean all
```

Comment:
put this action to line above with `&& yum clean all`. The smaller number of layers we have the faster the docker image build is